x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -196,30 +310,34 @@ elastic-agent run [global-flags]
 
 These flags are valid whenever you run `elastic-agent` on the command line.
 
-`-c <string>`::
+`--c <string>`::
 The policy file to use. If not specified, {agent} uses
-`{path.home}/elastic-agent.yml`.
+`{path.config}/elastic-agent.yml`.
+
+//REVIEWERS: Previously we decided that we would not document the debug selector
+//option (--d). Is that still true?
 
 `--e`::
 Log to stderr and disable syslog/file output.
 
 `--environment <environmentVar>`::
 The environment in which the agent will run.
 
-//TODO: Clarify what we mean by environment by showing an example.
-
 `--path.config <string>`::
 The directory where {agent} looks for its policy file. The default
 varies by platform.
 
+//REVIEWERS: In 7.10, aren't the installation paths hard-coded? If so, are the
+//path.* settings below still valid? They show up in the CLI help.
+
```

Comment:
They are still relevant when not using Agent from the `install` command. Which we still support, but it is very recommended to use the `install` command.

But for users that want to install from custom paths or run in some customer configuration it is useful to have.

Reply:
OK, but I feel like that we should document the recommended installation as the supported way to install the agent. I will leave the descriptions as-is.